hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SnapshotChainManager.java

```diff
@@ -246,7 +257,7 @@ private boolean deleteSnapshotPath(String snapshotPath,
     return status;
   }
 
-  private void loadFromSnapshotInfoTable(OMMetadataManager metadataManager)
+  public void loadFromSnapshotInfoTable(OMMetadataManager metadataManager)
```

Comment:
Add a javadoc for this now that it is marked `public`.

State that it clears the existing chain and reloads when called twice.

Reply:
It can be made `private`. I made it `public` because of previous changes. Now that change is modified we can make it `private` again